src/watch/index.ts

```diff
@@ -125,7 +126,7 @@ export class Task {
 
 		this.outputs = outputOptions;
 		this.outputFiles = this.outputs.map(output => {
-			return path.resolve(output.file || output.dir);
+			if (output.file || output.dir) return path.resolve(output.file || output.dir);
```

Comment:
In which situation does this if-statement trigger? We should probably adjust either the type of outputFiles to reflect that entries can be undefined or return something explicit  for other cases.